.github/workflows/dockerhub.yaml

```diff
@@ -0,0 +1,27 @@
+name: Publish Docker image
+
+on:
+  push:
+    branches: ["main"]
+
+jobs:
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3
+      
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+      
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v4
+        with:
+          context: .
+          file: ./Dockerfile.dockerhub
```

Comment:
would it make sense to rename `Dockerfile.dockerhub` => `Dockerfile` in a separate commit ?